src/mainwindow.cpp

```diff
@@ -29,14 +37,14 @@ MainWindow::~MainWindow()
 //Runs when Instruction Rate is selected by user
 void MainWindow::on_instructionRateButton_clicked()
 {
-    programName = relativePath + "instructionrate/" + isa + "instructionrate" + ext;
+    programName = binaryPath + "/" + relativePath + "instructionrate/" + isa + "instructionrate" + ext;
     spawnProcess();
 }
 
 //Runs when Memory Bandwidth is selected by user
 void MainWindow::on_memBandwidthButton_clicked()
 {
-    programName = relativePath + "MemoryBandwidth/" + isa + "MemoryBandwidth" + ext;
+    programName = binaryPath + "/" + relativePath + "MemoryBandwidth/" + isa + "MemoryBandwidth" + ext;
```

Comment:
Does Qt have a path builder we can use?

Reply:
Yes, I’ll look into how worthwhile it is.